Gruntfile.js

```diff
@@ -39,13 +39,21 @@ module.exports = function (grunt) {
 		nodeunit: {
 			all: 'test/node.js'
 		},
+		mochaTest: {
```

Comment:
> Are you saying you want mocha to run without running babel? If we do that, then we can't test es6 imports.

No, I am saying to run Babel without mocha, then we might be able to use ES6 imports, just dumping the test runner.